dockerfiles/Dockerfile.prod

```diff
@@ -1,6 +1,6 @@
 # node-build stage
 
-FROM docker-registry.default.svc:5000/openshift/node:16-slim AS node-build
+FROM node:20-slim AS node-build
```

Comment:
@jonespm I chose Node 20 since it is under LTS https://endoflife.date/nodejs.

CCM also is on Node 20 until rewrite done

Reply:
Also, @jonespm I have a question, in the Local Supervisor setup (supervisor_docker.conf) or package.json, there is no mention of Node version. So I am guessing it is picking up what ever based on the default set with supervisor?

Should Be set a Node version in supervisor_docker.conf? since that's how the FE package are build from

Reply:
Correct, it just runs whatever npm version is installed by this process. I don't think it needs a node version in there.